acceptance/tests/generic/generic_test.rb

```diff
@@ -0,0 +1,18 @@
+test_name 'test generic installers'
```

Comment:
for the file path for this test, can we put it in a folder that specifies that we're testing installers? Maybe `acceptance/tests/install/generic_test.rb`?